src/__tests__/index.tsx

```diff
@@ -92,21 +90,21 @@ export default ({children}) => <div className="sub-dir">{children}</div>
         <h1>
           This is the title
         </h1>
+        
+
+        import Demo from './demo'
+        
+
```

Comment:
@wooorm, I've switched this to use `xdm/esbuild.js` (which is great) and it looks like imports in MDX are not actually compiled to imports but are interpreted as strings and therefore end up in the final output. Any ideas why this is?

Reply:
Nevermind, I just had something weird locally. This is working!

Reply:
Trying things out again and I sure wish I'd taken a second to describe what the problem was because I've bumped into this again 😅